modules/auxiliary/scanner/http/totaljs_traversal.rb

```diff
@@ -0,0 +1,136 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
```

Comment:
This shouldn't be necessary.

```suggestion
```

Reply:
Sorry, I started from a template taken from the official documentation. I provide to fix